src/optionals/argparse/argparse.du

```diff
@@ -0,0 +1,132 @@
+import Argparse;
+import System;
+
+class Args {
+    init(private name, private desc) {}
+}
+
+class Parser {
+    private name;
+    private desc;
+    private userUsage;
+    private args;
+
+    var preArgs = [];
+    var required = [];
+
+    init(var name, var desc, var userUsage) {
+        this.args = Args(name, desc);
+    }
+
+    private flagExists(flag) {
+        for (var i = 0; i < this.preArgs.len(); i+=1) {
+            if (this.preArgs[i]['flag'] == flag) {
+                return true;\
```

Comment:
I see there's a couple of backslashes in here! This is my fault for the script not being documented at all but there is actually a [generate.du](https://github.com/dictu-lang/Dictu/blob/develop/scripts/generate.du) file that'll deal with creating these header files for you!